io/pci/dlpar.py.data/dlpar.yaml

```diff
@@ -1,4 +1,4 @@
-url: 
+manageSystem:
```

Comment:
why the new parameter is  `lssyscfg` has a bug? ,

Reply:
no, the old code written with assumption that lpar name will start with managed system.. which is the case for all our internal systems... but  it is not the ideal case for everyone.. so system name can change. it must be passed from user

Reply:
ah ok got it,